tests/unit/graph_learning_test.py

```diff
@@ -1661,18 +1727,36 @@ def test_5lm_feature_experiment(self):
         config = copy.deepcopy(self.feature_5lm_config)
         with MontyObjectRecognitionExperiment(config) as exp:
             pprint("...training...")
-            exp.train()
-
-            train_stats = pd.read_csv(
-                os.path.join(exp.output_dir, "train_stats.csv")
-            )
-            # The following check is brittle and depends on sensor arrangement. Leaving
-            # the rest of the test intact to detect run failures, but disabling checking
-            # of particular results.
-            # self.check_multilm_train_results(train_stats, num_lms=5, min_done=3)
+            objects = [self.fake_obs_learn, self.fake_obs_house_3d]
+            graph_modules = self.get_5lm_gm_with_fake_objects(objects)
```

Comment:
I would just call this `learning_modules` instead of `graph_modules` to avoid confusion